src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewQuartoProjectPage.java

```diff
@@ -253,6 +264,90 @@ protected void onUnload()
       super.onUnload();
       session_.persistClientState();
    }
+
+   @Override
+   protected void validateAsync(final NewProjectResult input,
+                                final OperationWithInput<Boolean> onValidated)
+   {
+      
+      final FileSystemItem projFile = FileSystemItem.createFile(input.getProjectFile());
+      final FileSystemItem projDir = projFile.getParentPath();
+      server_.stat(projDir.getPath(), new ServerRequestCallback<FileSystemItem>()
+      {
+         @Override
+         public void onResponseReceived(final FileSystemItem item)
+         {
+            // no file at this path -- safe for use
+            if (!item.exists())
+            {
+               onValidated.execute(true);
+               return;
+            }
+            
```

Comment:
Or should it just bail if the directory exist ? not consider "almost empty" directories (i.e. ones with only `.Rproj` files)